### PR TITLE
Added CustomGlobalExceptionHandler class

### PR DESCRIPTION
**Pull request by opopovych**
Added meaningful validation for existing DTO classes

### CODE REVIEW

**Review by Sanderfuck — CHANGES_REQUESTED** [review]
Where is CustomGlobalExceptionHandler? I see only added validation

File: src/main/java/mate/academy/dto/BookDto.java [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
     private Long id;
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
+    @UniqueElements
```
Comment (sokimaaa) [inline src/main/java/mate/academy/dto/BookDto.java]: @UniqueElements applies for collections only, so it Is not working here

Comment (sokimaaa) [inline src/main/java/mate/academy/dto/BookDto.java]: consider using @Isbn here

File: src/main/java/mate/academy/dto/BookDto.java [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.UniqueElements;
 
 @Data
 public class BookDto {
     private Long id;
+    @NotNull
```
Comment (sokimaaa) [inline src/main/java/mate/academy/dto/BookDto.java]: Why do you need validation for this object? Will you apply validation for this object?

Comment (opopovych) [reply src/main/java/mate/academy/dto/BookDto.java]: i did mistake, i don't use this object and i don't need validation for its.

**Review by Sanderfuck — APPROVED** [review]

**Pull request merged**